flink-ml-lib/src/main/java/org/apache/flink/ml/feature/vectorassembler/VectorAssembler.java

```diff
@@ -47,10 +47,15 @@
 
 /**
  * A Transformer which combines a given list of input columns into a vector column. Types of input
- * columns must be either vector or numerical value.
+ * columns must be either vector or numerical types. The elements assembled in the same column must
+ * have the same size. The operator deals with null values or records with wrong sizes according to
```

Comment:
The statement that "the elements must have the same size" seems not accurate, as elements can be null or vectors of different sizes when handleInvalid is set to keep.